src/pept-compare/methods.py

```diff
@@ -17,7 +17,7 @@ def read_files():
     dfs = []
     for filename in filenames:
         print("opening", filename)
-        df = pd.read_excel(filename)
+        df = pd.read_excel(filename, engine='openpyxl')
```

Comment:
I had to do this to get the program running on my machine.
See if it works like this for you. Don't forget to update the dependencies!